src/interface/templates.js

```diff
@@ -39,7 +39,7 @@ dict['alert-html'] =
 
 dict['newFile'] =
     '<h1>{{title}}</h1>' +
-    '<input type="text" placeholder="' + lg('name') + '"></input><br>' +
+    '<input type="text" class="filename-input" data-type="file" placeholder="' + lg('name') + '"></input><br>' +
```

Comment:
Could you explain this change?

Reply:
This way the data type is considered a file. Because I added the selector for `.filename-input` to the modal `onKeyUp` listener, when you press enter on the new file input it treats it similarly to if you had clicked the new file button.